src/SampleWebApp/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Dto.CountryDto.g.cs

```diff
@@ -11,18 +12,21 @@
 
 namespace SampleWebApp.Application.Dto;
 
-public record CountryKeyDto(System.Int64 keyId);
-
-/// <summary>
-/// The list of countries.
-/// </summary>
-public partial class CountryDto
+public class CountryKeyDto
 {
 
     /// <summary>
     ///  (Required).
     /// </summary>
+    [Key]
```

Comment:
we need to understand why? and use NoxDtoDatabaseConfigurator to do this

Reply:
As discussed on the call, get rid of the Key attribute, leave EntityKeyDto as a Record and return ChildDto with keys only from PostToOwnedEntity method